apps/src/applab/DesignToggleRow.jsx

```diff
@@ -69,13 +105,22 @@ module.exports = React.createClass({
     }
 
     return (
-      <div>
+      <div style={wrapperStyle} className="justify-contents">
+        <button
+            id='codeModeButton'
+            style={codeButtonStyle}
+            className='no-outline'
+            onClick={this.handleSetMode.bind(this, Mode.CODE)}>
+          {msg.codeMode()}
+        </button>
         <button
-          id="designModeToggle"
-          className="share"
-          onClick={this.handleModeToggle}>
-          { this.state.mode === Mode.DESIGN ? msg.codeMode() : msg.designMode() }
+            id='designModeButton'
+            style={designButtonStyle}
+            className='no-outline'
+            onClick={this.handleSetMode.bind(this, Mode.DESIGN)}>
+          {msg.designMode()}
         </button>
+        {' ' /* Needed for "text-align: justify;" to work. */ }
```

Comment:
Why do we have some of our styles in class form (no-outline, justify-contents). Is it because there's not an easy way to do :focus or :after inline?